MiniGamesBox Classic/src/main/java/plugily/projects/minigamesbox/classic/kits/basekits/Kit.java

```diff
@@ -122,8 +156,25 @@ public void setDescription(List<String> description) {
 
   public abstract ItemStack getItemStack();
 
-  public abstract void giveKitItems(Player player);
+  public void giveKitItems(Player player) {
+    player.getInventory().clear();
+    kitItems.forEach((item, slots) -> {
+      for (Integer slot : slots) {
+        switch (slot) {
+          case (-5): player.getInventory().setHelmet(item);
+          case (-4): player.getInventory().setChestplate(item);
```

Comment:
What happens if a user using this slot `-4` to set the boots for example or just a random itemStack rather than a chestPlate item?

Reply:
It would still put that item into that slot as the method forces it i'm pretty sure.